docs/services/general-info/envvar-scopes.md

```diff
@@ -34,14 +36,32 @@ Note that this envvar is the global representation of the local example from abo
 
 To get a list of global envvars used in all services, see the [Global Environment Variables](https://doc.owncloud.com/ocis/next/deployment/services/env-vars-special-scope.html#global-environment-variables) table in the ocis admin documentation.
 
-### Lifecycle
+## Lifecycle of Envvars
+
+The envvar struct tag contains at maximum the following key/value pairs to document the lifecycle of a config variable:
+
+* `introductionVersion`
+* `deprecationVersion`
+* `removalVersion`
+* `deprecationInfo`
+* `deprecationReplacement`
+
+### Introduce new Envvars
+
+If a new envvar is introduced, only the `introductionVersion` is required.
+
+{{< hint warning >}}
+During the development cycle, the value for the `introductionVersion` must be set to `%%NEXT%%`. This placeholder will be removed by the real version number during the production releasing process. 
+{{< /hint >}}
+
+For the documentation only to show the correct value for the `IV` (introduction version), our docs helper scripts will automatically generate the correct version to be printed in the documentation. If `%%NEXT%%` is found in the query, it will be replaced with `next`, else the value found is used.
```

Comment:
```suggestion
For the documentation to show the correct value for the `IV` (introduction version), our docs helper scripts will automatically generate the correct version to be printed in the documentation. If `%%NEXT%%` is found in the query, it will be replaced with `next`, else the value found is used.
```